### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by SFarida**
### This PR contains the following: 
- a file `app.rb` which contains methods for the following actions:

1. List all books.
2. List all people.
3. Create a person (teacher or student, not a plain Person).
4. Create a book.
5. Create a rental.
6. List all rentals for a given person id.

- a file `main.rb` to run the app

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      print 'Name: '
+      name = gets.chomp
+      if profession == 1
+        create_student(name, age)
+      else
+        create_teacher(name, age)
+      end
+      puts 'Person created successfully'
+      @option.list_options
+    else
```
Comment (pinkmoon25) [inline app.rb:39]: - [ ] Great work so far, however I think there is a bug that need your attention currently when we create a person whether it is teacher or student, the input we provide for parent_permission attribute for user and specialisation attribute for teacher comes out in the value of age when we list all people, I have recorder a short video demo below please see and try to fix this :point_down: 

https://user-images.githubusercontent.com/97521557/233792445-8d188d72-1fc5-4e0f-a5a4-e42310718273.mp4

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @SFarida , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
Rubocop Checks :heavy_check_mark: 
Gitflow :heavy_check_mark: 
app and main :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (SFarida) [reply app.rb:39]: Hello @PinkMoon25 thank you for your remark I will correct it now

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @SFarida ,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No linter errors :+1: 
- Good use of GitFflow :+1: 
## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**